### ISSUE TITLE
Request to make this repo private

### ISSUE THREAD

**Issue by ire-and-curses** [joelmcdaniel/stellar_exercise#1]
Hi Joel,

I'm writing to respectfully ask if you would be willing to move this repo to private visibility. We use these challenges to fairly assess candidates, and it would be better if those candidates were not a Google search away from a cut and paste solution.

**Comment by joelmcdaniel** [comment]
No, I'm sorry. Maybe you should have hired me and once again I am in need of a job. I worked for free to create this for you with the right intentions. Making it private for some time hasn't helped me, so it's going back to public. Hope you enjoyed my solution but not too much at this point.
